app/src/main/java/com/myfitnesstracker/dto/Nutrition.kt

```diff
@@ -0,0 +1,5 @@
+package com.myfitnesstracker.dto
+
+//Added Data Class to handle live data with all the variables I seen from the unit test
```

Comment:
Remove this comment; it's clutter.  The discussion you have in the pull request is adequate.